src/Metadata/PropertyMetadata.php

```diff
@@ -39,6 +41,9 @@ public function __construct(
         if ($accessor) {
             $this->setAccessor($accessor);
         }
+        foreach ($customInformation as $key => $value) {
+            $this->setCustomInformation($key, $value);
```

Comment:
This could fail hard if `$key` is not a string, maybe a check or cast is needed.

Reply:
right

Reply:
only reason this would be non-string is when its something that could be a number. added cast to string.